internal/handler/beer.go

```
@@ -38,13 +38,18 @@ func (h WorkspaceServer) HandleBeerListPage(reqResp *web.ReqRespPair) error {
 	if countryErr != nil {
 		return apperr.NewBadRequestError("Invalid country", countryErr)
 	}
+	withoutSlot, withoutSlotErr := reqResp.GetBoolQueryParam("withoutSlot")
+	if withoutSlotErr != nil {
+		return apperr.NewBadRequestError("Invalid withoutSlot", withoutSlotErr)
+	}
 
 	page := layout.Page{Title: fmt.Sprintf("Beer Workspace")}
```

golangci-lint (2.5.0)

[error] S1039: unnecessary use of fmt.Sprintf

(staticcheck)